Signal/src/ViewControllers/OWSConversationSettingsTableViewController.m

```diff
@@ -168,31 +163,50 @@ - (void)updateTableContents
     firstSection.customHeaderHeight = @(100.f);
 
     if (!self.isGroupThread && self.thread.hasSafetyNumbers) {
-        [firstSection addItem:[OWSTableItem itemWithCustomCellBlock:^{
-            UITableViewCell *cell = [UITableViewCell new];
-            cell.accessoryType = UITableViewCellAccessoryDisclosureIndicator;
```

Comment:
I think doing so allowed the "actionBlock" to be brought left a bit.

clang-format is a whimsical beast.